sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,507 @@
+                    and disbursement_overawards.disbursement_value_code = ''BCSL''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "BCSL Overawards Balance",
+            students.disability_status as "Permanent Disability status",
+            -- Application Personal Info (does not include Institution or Program info)
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
+            applications.data ->> ''citizenship'' as "Citizenship status",
+            applications.data ->> ''bcResident'' as "BC Residency Status",
+            applications.data ->> ''indigenousStatus'' as "Indigenous Person status",
+            applications.data ->> ''aboriginalHeritage'' as "Indigenous category code",
+            applications.data ->> ''applicationPDPPDStatus'' as "PD/PPD Application Flag",
+            applications.data ->> ''youthInCare'' as "Youth in Care Flag",
+            ''--TBD--'' as "Youth in Care beyond age 19 Flag",
+            applications.data ->> ''outOfHighSchoolFor4Years'' as "Outside of highschool >4 years flag",
+            applications.data ->> ''whenDidYouGraduateOrLeaveHighSchool'' as "High School Graduation Date",
+            applications.data ->> ''fulltimelabourForce'' as "Recent Labour Force Participation Flag",
+            applications.data ->> ''addTrustContactToContactSABC'' as "Trusted Contact Identified (Yes/No)",
+            applications.data ->> ''relationshipStatus'' as "Marital Status",
+            --Application Financial Info
+            applications.data ->> ''taxReturnIncome'' as "Total income",
+            applications.data ->> ''partTimeAwardTypesToBeConsidered'' as "Grant-only flag",
+            applications.data ->> ''scholarshipsreceviedCosts'' as "Merit-based scholarship amount",
+            applications.data ->> ''parentvoluntarycontributionsCosts'' as "Voluntary parental contribution amount",
+            applications.data ->> ''governmentFundingCosts'' as "Government funding amount",
+            applications.data ->> ''nongovernmentfundingCosts'' as "Non-government funding amount",
+            applications.data ->> ''bcIncomeassistancefordisabilitiesCosts'' as "B.C. Income Assistance Amount",
+            applications.data ->> ''returntriphomeCost'' as "Relocation costs",
+            -- Application Financial Appeals Info
+            applications.data ->> ''exceptionalExpensesApplicationException'' as "Exceptional expense amount",
+            applications.data ->> ''daycareCosts11YearsOrUnder'' as "Daycare cost amount",
+            applications.data ->> ''totalUnsubsidizedDayCareCostsDuringClassHoursForTheProposedStudyPeriodForDisabledInfirmedDependents12YearsOfAgeAndOlderPanel'' as "Unsubsidized daycare amount",
+            ''--TBD--'' as "Child/Spousal Support amount",
```

Comment:
Is this the child/spousal support amount? 

![image](https://github.com/bcgov/SIMS/assets/54600590/76218b9f-d7f6-4f07-bf85-ef9442e9d0e4)